integration/cleanup/cleanup.setup.ts

```diff
@@ -5,20 +5,19 @@ import { test as setup } from '@playwright/test';
 import { appConfigs } from '../presets/';
 
 setup('cleanup instances ', async () => {
-  const entries = Object.values(appConfigs.envs)
```

Comment:
`appConfig.envs` didn't contain all the keys in use, so not all instances were getting cleaned up as far as I could tell.